fda-model/metrics/Results.cs

```diff
@@ -20,33 +23,34 @@ public class Results: IContainResults
         }
         public Results(){
             _aepThreshold = 0.0;
-            _meanAEP = 0.0;
-            _aepCount = 0;
-            _meanEADs = new Dictionary<string, double>();
-            _eadCounts = new Dictionary<string, Int64>();
+            _aep = new Histogram(null, AEP_HISTOGRAM_BINWIDTH);
+            _ead = new Dictionary<string, Histogram>();
         }
         public void AddAEPEstimate(double aepEstimate)
         {
-            _meanAEP = _meanAEP +((aepEstimate - _meanAEP)/(double)_aepCount);
-            _aepCount +=1;
+            double[] data = new double[1] { aepEstimate };
```

Comment:
check to see if _aep is null, if so, construct it here with the first estimate. (much like you do with EAD below.